api/v1alpha1/amaltheasession_children.go

```diff
@@ -120,6 +120,33 @@ func (cr *AmaltheaSession) StatefulSet() (appsv1.StatefulSet, error) {
 		TerminationMessagePolicy: v1.TerminationMessageReadFile,
 	}
 
+	// Assign a readiness probe to the session container
+	switch cr.Spec.Session.ReadinessProbe.Type {
+	case HTTP:
+		sessionContainer.ReadinessProbe = &v1.Probe{
+			ProbeHandler: v1.ProbeHandler{
+				HTTPGet: &v1.HTTPGetAction{
+					Port: intstr.FromInt32(cr.Spec.Session.Port),
+					Path: cr.Spec.Session.URLPath,
```

Comment:
Note: this is probably something that should be an option for clients. The health check path may not be the session path.

Reply:
Yes I agree. We could add a `custom` readiness probe type that will allow people to fully define all these things. But for now we dont need it so we can add it later if needed.